### DISCUSSION TITLE
Any equivalence of keyring encryption for Kong in db-less mode?

### DISCUSSION THREAD

**Discussion by shanexguo** [Kong/kong#9439] [Help] [answered]
We are migrating to Kong db-less mode, and it seems our existing keyring encryption feature won't work in db-less mode. Any suggestions for its replacement? Thanks,

**Comment by fffonion** [comment, marked as answer]
@shanexguo you will be able to use secrets management, which reaches GA in coming 3.0 release. https://docs.konghq.com/gateway/latest/kong-enterprise/secrets-management/

**Comment by shanexguo** [comment]
Thanks a lot for the tip. Will wait for 3.0 and test it out.
